lighthouse-core/audits/seo/font-size.js

```diff
@@ -8,10 +8,26 @@
 /** @typedef {LH.Artifacts.FontSize['analyzedFailingNodesData'][0]} FailingNodeData */
 
 const URL = require('../../lib/url-shim');
+const i18n = require('../../lib/i18n/i18n.js');
 const Audit = require('../audit');
 const ViewportAudit = require('../viewport');
 const MINIMAL_PERCENTAGE_OF_LEGIBLE_TEXT = 60;
 
+const UIStrings = {
+  /** Imperative title of a Lighthouse audit that tells the user that they should use font sizes that are easily read by the user. This is displayed in a list of audit titles that Lighthouse generates. */
+  title: 'Document uses legible font sizes',
+  /** Imperative title of a Lighthouse audit that tells the user that they should use font sizes that are easily read by the user. This imperative title is shown to users when there is a font that is too small to be read by the user. */
+  failureTitle: 'Document doesn\'t use legible font sizes',
+  /** Description of a Lighthouse audit that tells the user *why* they need to use a larger font size. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'Font sizes less than 12px are too small to be legible and require mobile ' +
+      'visitors to “pinch to zoom” in order to read. Strive to have >60% of page text ≥12px. ' +
+      '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/font-sizes).',
+  /** [ICU Syntax] Label for the audit identifying font sizes that are too small. */
+  displayValue: '{percentFormatted, number, percent4SigFig} legible text',
```

Comment:
is there a reason we're upping it to 4 instead of leaving at 1 decimal point?

Reply:
The 4 sigfig percent will make it form XX.XX%, so it matches to two trailing decimal places, I upped it from 1 place to match the failing legible text percent which is formatted to 2 decimal places
`percentageOfFailingText = parseFloat((100 - percentageOfPassingText).toFixed(2));`
It seemed like they should match.

Reply:
oh, not it in diff but elsewhere in the file gotcha! my b 👍

To explain my multi-comment review just about the single decimal place since at this point it seems like it needs explaining not as random hate on your PR (😄): 

In general, I'm a fan of less precision in Lighthouse UI. Especially cases where we can be somewhat hand-wavy to begin with in the impl, or there's really no action to be taken based on the additional precision.

Reply:
sorrrrrry to keep going on this.....but can we actually rename the variable then too? `percentFormatted` makes me think we are passing it a `percentFormatted` string when that's what's going to come out on the other side, or am I misunderstanding the point of the ICU syntax names?